plugins/stylescombo/plugin.js

```diff
@@ -6,6 +6,30 @@
 ( function() {
 	'use strict';
 
+	function getCutOffIndex(elements, tagNames) {
+		// given an array of elements and an array of tag names to stop on,
+		// returns the earliest index of any of the tag names, or -1
+
+		var minIndexes = {},
+			cutOffIndexes = [];
+
+		for (var i = elements.length - 1; i >= 0; --i) {
+			minIndexes[elements[i].getName()] = i;
+		}
+
+		tagNames.forEach(function(tagName, _, __) {
+			if (minIndexes[tagName] !== undefined) {
```

Comment:
I had seen that around, but never actually "learned" it that way and was wondering why.  So I did some looking into it and found this, which catches me up on the state of things and also reveals some hilarious JavaScript things that I did not realize were a thing.  After reading this I think I will keep it as-is.  

http://www.2ality.com/2013/04/check-undefined.html